sdk/search/Azure.Search.Documents/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0012:Avoid single word type names", Justification = "<Pending>", Scope = "type", Target = "~T:Azure.Search.Documents.Models.Autocompletion")]
```

Comment:
I've added suppressions so the build passes. I suspect you'll be addressing these during the API review.